lib/invisible_captcha/controller_ext.rb

```diff
@@ -101,6 +101,9 @@ def honeypot_spam?(options = {})
           if params[default_honeypot].present?
             warn_spam("Invisible Captcha honeypot param '#{default_honeypot}' was present.")
             return true
+          elsif params.dig(scope, default_honeypot).present?
```

Comment:
To make it more consistent, what about adding this check in the IF branch? It will be quite similar to this line:

https://github.com/markets/invisible_captcha/blob/ba9e5976d096dba42cae634e02be7afae7912d03/lib/invisible_captcha/controller_ext.rb#L91

Reply:
refactored to be more consistent 👍